client/lib/js/react/components/asset_action_panel.js

```diff
@@ -44,7 +35,7 @@ const AssetActionPanel = React.createClass({
                     {accountList.map((account) => (
                         <MenuItem
                             key={account.name}
-                            onClick={() => this.handleAccountClick(account)}>
+                            onClick={() => handleAccountClick(account)}>
```

Comment:
Ah, now you need a `safeInovke` here, or default it to `noop`, or use `isRequired`.